contribs/gmf/less/layertree.less

```diff
@@ -34,10 +36,10 @@
       color : @color-light;
       display: inline-block;
       &::before {
-        content: "\f054";
+        content: @fa-var-chevron-right;
```

Comment:
Awesome. I didn't know there were variable for the unicode codes in font-awesome. Nice catch.